samples/apps/copilot-chat-app/webapp/src/components/chat/ChatWindow.tsx

```diff
@@ -98,14 +100,28 @@ const useClasses = makeStyles({
 export const ChatWindow: React.FC = () => {
     const classes = useClasses();
     const dispatch = useAppDispatch();
+    const { instance, inProgress } = useMsal();
+    const msGraph = useGraph();
+    const chatService = new ChatService(process.env.REACT_APP_BACKEND_URI as string);
+    const { alerts } = useAppSelector((state: RootState) => state.app);
+
     const { conversations, selectedId } = useAppSelector((state: RootState) => state.conversations);
     const chatName = conversations[selectedId].title;
+    const chatUsers = conversations[selectedId].users;
+
     const [title = '', setTitle] = useState<string | undefined>(selectedId);
     const [isEditing, setIsEditing] = useState<boolean>(false);
-    const { instance, inProgress } = useMsal();
-    const { alerts } = useAppSelector((state: RootState) => state.app);
 
-    const chatService = new ChatService(process.env.REACT_APP_BACKEND_URI as string);
+    useEffect(() => {
+        if (!conversations[selectedId].userDataLoaded) {
+            void msGraph.loadUsers(chatUsers.map((user: IChatUser) => user.id)).then(() => {
```

Comment:
nit: `void` seems unnecessary.

Reply:
This is necessary to adhere to linting rules @dehoward just enabled 
![image](https://github.com/microsoft/semantic-kernel/assets/125500434/92ce32c7-ef28-4406-bd57-7dcd35a550c9)

Reply:
Can you call catch on the promise at the end: https://typescript-eslint.io/rules/no-floating-promises/?

Reply:
A catch would be redundant here because all the business logic is in the try block inside the `loadUsers` function, so any errors are caught and handled appropriately.  It makes more sense to keep the error handling contained there rather than where the function is invoked